app/assets/stylesheets/action_page.scss

```diff
@@ -373,21 +377,30 @@
   > div {
     text-align: center;
   }
-  .tool {
+  .go-back {
+    margin: 1rem 0 3rem;
     #tweet-back {
-      text-decoration: underline;
+      @extend .eff-button;
+      background: $gray-light;
+      border-bottom: 0;
+      clear: both;
+      color: $charcoal;
+      float: left;
+      font-size: $base-font-size * 0.8;
+      margin: 1rem auto;
+      max-width: 100%;
+      text-transform: none;
       &:hover {
-        cursor: pointer;
+        background: $gray-light;
+        border-bottom: 0;
+        color: $charcoal;
+        cursor: default;
```

Comment:
can the cursor be a pointer again? it's not super obvious that it's clickable to me

Reply:
No, that's the thing, it's not clickable.

Reply:
clicking it brings you back to the take action tool though

Reply:
I've lost my damn mind.